sys/net/gnrc/network_layer/sixlowpan/frag/sfr/gnrc_sixlowpan_frag_sfr.c

```diff
@@ -444,28 +464,42 @@ int gnrc_sixlowpan_frag_sfr_forward(gnrc_pktsnip_t *pkt,
                           page);
 }
 
+static int _report_non_ack_req_window_sent(clist_node_t *node, void *fbuf_ptr)
+{
+    _frag_desc_t *frag_desc = (_frag_desc_t *)node;
+    if (!_frag_ack_req(frag_desc)) {
+        gnrc_sixlowpan_frag_sfr_congure_snd_report_frag_sent(fbuf_ptr);
+    }
+    return 0;
+}
+
 void gnrc_sixlowpan_frag_sfr_arq_timeout(gnrc_sixlowpan_frag_fb_t *fbuf)
 {
-    uint32_t now = xtimer_now_usec();
-    _frag_desc_t * const head = (_frag_desc_t *)fbuf->sfr.window.next;
-    _frag_desc_t *frag_desc = head;
+    uint32_t now = xtimer_now_usec() / US_PER_MS;
+    _frag_desc_t *frag_desc = (_frag_desc_t *)fbuf->sfr.window.next;
     uint32_t next_arq_offset = fbuf->sfr.arq_timeout;
     bool reschedule_arq_timeout = false;
     int error_no = ETIMEDOUT;   /* assume time out for fbuf->pkt */
 
     DEBUG("6lo sfr: ARQ timeout for datagram %u\n", fbuf->tag);
     fbuf->sfr.arq_timeout_event.msg.content.ptr = NULL;
+    if (IS_USED(MODULE_GNRC_SIXLOWPAN_FRAG_SFR_CONGURE) && frag_desc) {
+        gnrc_sixlowpan_frag_sfr_congure_snd_report_frags_timeout(fbuf);
+        _shrink_window(fbuf);
+        frag_desc = (_frag_desc_t *)fbuf->sfr.window.next;
+    }
+    _frag_desc_t * const head = frag_desc;
```

Comment:
Please add a short comment what's going on here.

Reply:
I'll move up the comment below... other than that, this just moves the old variable names to the new, congure-based names.